content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -406,9 +406,12 @@ Note that if this update would have been an `Apply` operation, the operation wou
 
 #### Conflicts
 
-A conflict is a special status error that occurs when an `Apply` operation tries to change a field, which another user also claims to manage.
+A conflict is a special status error that occurs when an `Apply` operation tries to change a field, which another user also claims to manage. This will prevent an applier from unintentionally overwriting the value set by another user. When this occurs, the applier has 3 options to resolve the conflicts:
+
+* If overwriting the value was intentional (or if the applier is an automated process like a controller) the applier should set the `force` query parameter to true and make the request again. This will force the operation to succeed and the field will be removed all other field managers that claimed to manage it.
```

Comment:
I'm not completely sure what's going to happen when you say "This will force the operation to succeed and the field will be removed all other field managers that claimed to manage it."

Reply:
Ah yeah, that is kind of ambiguous